web/js/treedialog.js

```diff
@@ -230,61 +219,67 @@ function cancelDebug() {
 function initTreeDialog(pic) {
   fullPic = pic;
 
-  const div = document.createElement('div');
-  dialog = $(div).dialog({
+  const $container = $('<div>');
+  $dialog = $container.dialog({
```

Comment:
I am confused.  The point of this was to get rid of the popup window.  It looks like you're still creating one.

Reply:
There is no popup window anymore as the dialog is contained in the area right below canvas. Should I create a custom element rather than reusing jQuery's dialog?

Reply:
Okay, will do.